webapp/channels/src/components/widgets/popover/index.tsx

```diff
@@ -21,30 +21,23 @@ interface Props {
     onMouseOver?: React.MouseEventHandler<BSPopover>;
 }
 
-export default class Popover extends React.PureComponent<Props> {
-    static defaultProps = {
-        placement: 'right',
-        popoverStyle: 'info',
-        popoverSize: 'sm',
+const Popover = ({placement = 'right', popoverSize = 'sm', children, popoverStyle = 'info', title, id, onMouseOut, onMouseOver, className, style}: Props) => {
```

Comment:
To improve readability, keep one Prop per line:

```suggestion
const Popover = ({
    placement = 'right',
    popoverSize = 'sm',
    children,
    popoverStyle = 'info',
    title,
    id,
    onMouseOut,
    onMouseOver,
    className,
    style,
}: Props) => {
```